### DISCUSSION TITLE
Amazing

### DISCUSSION THREAD

**Discussion by BlackDahlia313** [stemplayer-js/stemplayer-js#1] [General]
I just wanted to share my gratitude towards this project. This is amazing and the demos are just perfect. 

Keep up the awesome work! <3

**Comment by mpcremer** [comment]
> I just wanted to share my gratitude towards this project. This is amazing and the demos are just perfect. 
> 
> 
> 
> Keep up the awesome work! <3

Great! Glad to know you like the player... (and very kind of you to take the trouble to let us know!) 😄

**Comment by BlackDahlia313** [comment]
@mpcremer hey! could we get a new beta release so we can test around the new region lock etc?

**Comment by mpcremer** [comment]
done 

Mark Cremer

Managing Director & Principal Software Consultant
First Coders LTD
www.firstcoders.co.uk

**Reply by BlackDahlia313** [reply]
You are my sunshine in this cold Detroit weather. ty sir

**Comment by mpcremer** [comment]
haha.. i couldnt possibly aim for anything more worthwhile!

Mark Cremer

Managing Director & Principal Software Consultant
First Coders LTD
www.firstcoders.co.uk
